common/io/Serial.cpp

```diff
@@ -170,8 +171,6 @@ bool OpenAndFlock(const std::string &path, int oflag, int *fd) {
   // further opens.
   if (ioctl(*fd, TIOCEXCL) == -1) {
     OLA_WARN << "TIOCEXCL " << path << " failed: " << strerror(errno);
-    close(*fd);
```

Comment:
Shouldn't we still close to tidy up?

Reply:
It now closes (and unlocks, if necessary) on all exit paths.